bundle-workflow/src/ci_workflow/ci_target.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+
+class CiTarget:
+    version: str
```

Comment:
Just confirming, won't we need `arch` as a part of `CiTarget`?

Reply:
Nope, it doesn't use arch anywhere.